src/styles/calendar.scss

```diff
@@ -0,0 +1,49 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-size;
+  width: $calendar-size;
+  padding: 10px;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition: all;
+    transition-duration: 0.5s;
```

Comment:
The 'transition' property is used twice here, once with a shorthand (line 12) and once with 'transition-duration' (line 13). It's best to consolidate these into a single shorthand property to keep the code clean and avoid potential overrides or conflicts.